src/templates/home_page.hbs

```diff
@@ -17,9 +17,14 @@
   </div>
 </section>
 
-<div class="toggle-container">
-  <button id="toggle-categories" class="categories-toggle-btn">Switch to Private</button>
-</div>
+{{#if signed_in}}
+  {{#each (filter categories on="description" starts_with="Internal knowledge base for the Support")}}
+  <!-- toggle button will only appear if there is a category whose description starts with internal AND the user is logged in -->
```

Comment:
Do handlebar comments work?  If so, that'd be preferred over html comments because they won't be rendered.
```
{{!-- Some comment --}}
```

Reply:
@christophermluna

Reply:
it's fine to switch this to handlebar comments.

Reply:
Thanks, just committed the changes.